web/template/layout/identity-details.gohtml

```diff
@@ -1,19 +1,14 @@
 {{ define "identity-details" }}
-    {{ $fullName := (printf "%s %s" .Provided.IdentityUserData.FirstNames .Provided.IdentityUserData.LastName) }}
-
     <div class="govuk-summary-card">
         <div class="govuk-summary-card__title-wrapper">
             <h2 class="govuk-summary-card__title">{{ tr .App "yourConfirmedIdentityDetails" }}</h2>
```

Comment:
The [figma](https://www.figma.com/design/FCBLKeHWjTXcCHUj1JOZKR/Identity-flow-%232?node-id=863-213&node-type=frame&t=Kp3wbo5leyXfF7PG-0) for this design has dropped `Your` from this title. May be worth checking with design on how they want this to read.